notification-settings/lib/notification_settings/notification_library.rb

```diff
@@ -106,7 +91,7 @@ def local_settings_allow_push?(pusher)
       end
 
       def global_settings_allow_push?
-        target.notification_setting.settings.dig(:index)
+        target.notification_setting.settings.fetch(:index, true)
```

Comment:
I agree, it would make sense to change this to something more self-explanatory. What about `:pusher_disabled`?

And then it would make sense to also add this to the documentation: https://github.com/jonhue/notifications-rails/tree/master/notification-settings#pusher-specific-settings

Reply:
How about `:allow_push` (to parallel `global_settings_allow_push?`)?

Or `:pusher_enabled`? I'm reluctant to go with `:pusher_disabled` because I'd want to be consistent with the logic of the other settings, where `true` means it's enabled, not disabled...

Reply:
I pushed up a change/doc change with `:pusher_enabled`. I assume we won't worry about a migration for settings yet since I'm the only one using it... :smile: